File: app/src/main/java/com/example/calculator/MainActivity.java
package com.example.calculator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.calculator.databinding.ActivityMainBinding;
import com.example.calculator.layouts.Calculator;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);

        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());

        Calculator.init(binding);
        setContentView(binding.getRoot());

    }

}

File: kotlincalculator/src/main/java/com/example/kotlincalculator/MainActivity.kt
package com.example.kotlincalculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.kotlincalculator.databinding.ActivityMainBinding
import com.example.kotlincalculator.layouts.Calculator

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)

        val binding = ActivityMainBinding.inflate(layoutInflater)

        Calculator(binding)

        setContentView(binding.root)

    }
}